### PR TITLE
Data layer readme

### PR DESCRIPTION
**Pull request by bre1470**
To-do's:
- [x] Describe what DataTable is and how to use
- [x] DataOnDemand - what it is and how to use
- [x] How to initialize data source
- [x] How to use data for Highcharts
- [x] How to use data for DataGrid
- [x] Add description  for each directory: converters / modifiers / stores

### CODE REVIEW

File: ts/Data/README.md [hunk 1]
```diff
@@ -229,3 +229,43 @@ catch (error) {
...
+
+DataOnDemand is one of possible way to coordinate and share stores and their
+data between multiple modules. You can request the store multiple times, while
+the class will load each store only ones.
+
+```TypeScript
+const googleStore1 = await onDemand.getStore('My Google Spreadsheet');
+const googleStore2 = await onDemand.getStore('My Google Spreadsheet');
+const googleStore3 = await onDemand.getStore('My Google Spreadsheet');
+```
```
Comment (pawellysy) [inline ts/Data/README.md:334]: Maybe we could include an example, of how the store is connected to Dashboard?

**Review by pawellysy — COMMENTED** [review]
Seems like a good description. One comment tho

File: ts/Data/README.md [hunk 2]
```diff
@@ -4,3 +4,331 @@ Data Layer
...
+    }]
+});
+```
+
+
+
+
+### DataTable in DataGrid
+
+DataGrid visualizes and optionally modifiesication of cell content in a table.
```
Comment (karolkolodziej) [inline ts/Data/README.md:164]: This sentence is kind of odd...
`modifiesication`?

**Review by karolkolodziej — APPROVED** [review]
Good job!

File: ts/Data/README.md (hunk 1 above)
Comment (bre1470) [reply ts/Data/README.md:342]: Actually this is something that should be part of the Dashboard documentation.

**Review by pawellysy — APPROVED** [review]

**Pull request merged**